main.py

```diff
@@ -1,10 +1,40 @@
+import json
 import init_django_orm  # noqa: F401
-
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+
+    for nickname, player_data in players_data.items():
+        race_info = player_data.get("race")
+        race_player, created = Race.objects.get_or_create(
+            name=race_info.get("name"),
+            description=race_info.get("description")
+        )
+        skills = race_info.get("skills")
```

Comment:
Ensure that 'skills' is not None before iterating over it. Consider adding a check like 'if skills:' to avoid potential errors if 'skills' is missing or None.